tuning/data/data_handlers.py

```diff
@@ -328,57 +366,72 @@ def tokenize(
 
 def duplicate_columns(
     element: Union[Dict[str, str], Dict[str, List]],
-    old_column: str,
-    new_column: str,
+    old_column_name: str,
+    new_column_name: str,
     **kwargs,
 ):
-    """Function (data handler) to duplicate one columne of a dataset to another.
+    """Function to duplicate one columne of a dataset to another.
        Expects to be run as a HF Map API function.
     Args:
         element: the HF Dataset element
-        old_column: Name of the column which is to be duplicated
-        new_column: Name of the new column where duplicated column is to be saved
+        old_column_name: Name of the column which is to be duplicated
+        new_column_name: Name of the new column where duplicated column is to be saved
     Returns:
-        Formatted HF Dataset element with new_column where existing_columns content is deep copied.
+        Formatted HF Dataset element with
+        {"new_column_name": copy.deepcopy("old_column_name")}.
     """
-    if not old_column or not new_column:
+    if not old_column_name or not new_column_name:
         raise ValueError(
             "for duplicating columns both old and new column name must be specified"
         )
-    if old_column not in element:
+    if old_column_name not in element:
         raise ValueError(
-            f"Cannot duplicate {old_column} to {new_column} as column {old_column} doesn't exist"
+            "Cannot duplicate %s to %s as column %s doesn't exist"
+            % (old_column_name, new_column_name, old_column_name)
         )
-    if new_column in element:
+    if new_column_name in element:
         raise ValueError(
-            f"Cannot duplicate {old_column} to f{new_column} as column {new_column} already exists"
+            "Cannot duplicate %s to %s as column %s already exist"
+            % (old_column_name, new_column_name, old_column_name)
         )
 
     return {
-        f"{old_column}": element[old_column],
-        f"{new_column}": copy.deepcopy(element[old_column]),
+        f"{old_column_name}": element[old_column_name],
+        f"{new_column_name}": copy.deepcopy(element[old_column_name]),
```

Comment:
Is deepcopy the only way? If it is a data frame operation, won't it bepre efficient?